extensions/narayana-lra/runtime/src/main/java/io/quarkus/narayana/lra/runtime/NarayanaLRAProducers.java

```diff
@@ -16,7 +15,6 @@ public LRAParticipantRegistry lraParticipantRegistry() {
 
     @Produces
     public ParticipantProxyResource participantProxyResource() {
-        Object proxy = CDI.current().select(ParticipantProxyResource.class);
```

Comment:
This variable is unused. Not sure why you did that or if it's just a leftover?

Reply:
Thanks